src/Signup.jsx

```diff
@@ -0,0 +1,184 @@
+                <img
+                  src="/Peerwize Logo.png"
+                  alt="logo"
+                  height="127.08px"
+                  width="127.09px"
+                />
+
+                <p className="text-[#FBA04B] font-[700] text-[16px">
+                  connecting skills, creating futures...
+                </p>
+              </div>
+            </div>
+          </div>
+
+          {/* FORM DIV */}
+
+          <div className="form-scroll">
+            <div className=" grid gap-4 lg:gap-8">
+              <h1 className="text-[28px] font-[700]">
+                Sign up on <span className="text-[#37BBCA]">Peerwize</span>
+              </h1>
+
+              <div className="text-[16px] flex items-center gap-[8px] font-[400] justify-center">
+                <span className="signup_span" />
+                <p>
+                  Already have an account?{" "}
+                  <Link to="/" className="text-[#37BBCA]">
+                    {" "}
+                    {/* PATH TO LOGIN PAGE */}
+                    Login
+                  </Link>
+                </p>
+                <span className="signup_span" />
+              </div>
+            </div>
+
+            <form action="" className="py-4 pt-8 flex flex-col gap-y-6">
+              <input
+                type="email"
+                id="email"
+                name="email"
+                placeholder="Email"
+                className="auth_input"
+              />
+
+              <input
+                type="text"
+                id="firstName"
+                name="firstName"
+                placeholder="First name"
+                className="auth_input"
+              />
+
+              <input
+                type="text"
+                id="LastName"
+                name="LastName"
+                placeholder="Last name"
+                className="auth_input"
+              />
+
+              <select
+                name="country"
+                id="country"
+                className="auth_input text-[#848484] capitalize"
+              >
+                {Countries.map((country, index) => (
+                  <option value={country} key={index}>
+                    {country}
+                  </option>
+                ))}
+              </select>
+
+              <select
+                name="city"
+                id="city"
+                className="auth_input text-[#848484] capitalize"
+              >
+                {Cities.map((city, index) => (
+                  <option value={city} key={index}>
+                    {city}
+                  </option>
+                ))}
+              </select>
+
+              <input
+                type="number"
+                id="phoneNumber"
+                name="phoneNumber"
+                placeholder="Phone number"
+                className="auth_input"
+              />
+
+              <div className="relative">
```

Comment:
A good approach. But why not give this outer div a border and remove the border from the input that is a child of it? That way you won't need to use relative/absolute positioning.